tests/actions

```diff
@@ -379,6 +379,56 @@ class ActionsTestCase(faftests.DatabaseCase):
         bosra = self.db.session.query(BuildOpSysReleaseArch).count()
         self.assertEqual(bosra, init_bosra + 2)
 
+    def test_clenup_unassigned(self):
+        self.test_assign_release_to_builds()
+
+        # add package and lob that will not be deleted
+        pkg_stay = Package()
+        pkg_stay.build = self.db.session.query(Build).first()
+        pkg_stay.arch = self.db.session.query(Arch).first()
+        pkg_stay.name = "pkg-test-stay"
+        self.db.session.add(pkg_stay)
+        self.db.session.flush()
+
+        config["storage.lobdir"] = "/tmp/faf_test_data/lob"
```

Comment:
Can you  replace this (and the other occurence) with `tempfile.mkdtemp(prefix="faf")`?